Package.swift

```diff
@@ -51,7 +51,10 @@ let package = Package(
   ],
   targets: [
     .target(
-      name: "FBLPromises"
+      name: "FBLPromises",
+      resources: [
+        .copy("Sources/Resources/PrivacyInfo.xcprivacy")
```

Comment:
```suggestion
        .process("Sources/Resources/PrivacyInfo.xcprivacy")
```

nit: use `.process` instead of `.copy`. While I verified and it doesn't make a difference when generating the Privacy Report, [Apple's docs](https://developer.apple.com/documentation/PackageDescription/Resource/process(_:localization:)#:~:text=If%20possible%2C%20use%20this%20rule%20instead%20of) recommend `.process` over `.copy`.

Reply:
Could you also please do this for the `Promises` target on line 85?

Reply:
I assumed that since the `Promises` target depends on the `FBLPromises` target, the resource only needs to be included in the `FBLPromises` target. Is that correct?

Reply:
I agree, and think this is likely fine. However, out of an abundance of caution, I think it's safer to include the manifest again for the `Promises` target. Since Apple's list calls out both the `FBLPromises` and `Promises` modules, I think both should explicitly have one (even if the contents are the same/empty). That way, if there is tooling to enforce any module on the list has a direct manifest (as opposed to a transitive one), all cases are covered.

Reply:
Sounds good, added.